docker/php.Dockerfile

```diff
@@ -33,3 +36,4 @@ RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
 ## APACHE
 RUN a2enmod rewrite
 COPY docker/000-default.conf /etc/apache2/sites-available/000-default.conf
+COPY docker/php.ini /usr/local/etc/php/php.ini
```

Comment:
I'm surprised to see this change. Why was it needed?

Reply:
Mainly because we want to override some configurations for php.ini
Specifically we need to increase the file size limit.